src/node.js

```diff
@@ -178,12 +178,27 @@
   }
 
   startup.setupProcessObject = function() {
-    process._setupProcessObject(setPropByIndex);
+    const _hrtime = process.hrtime;
+    const hrValues = new Uint32Array(3);
 
-    function setPropByIndex() {
+    process._setupProcessObject(pushValueToArray);
+
+    function pushValueToArray() {
       for (var i = 0; i < arguments.length; i++)
         this.push(arguments[i]);
     }
+
+    process.hrtime = function hrtime(ar) {
+      const ret = new Array(2);
```

Comment:
If you use `[0, 0]` here, it should give V8 a hint that it's a numbers-only array.